azurelinuxagent/ga/update.py

```diff
@@ -261,6 +261,9 @@ def run(self):
             exthandlers_handler = get_exthandlers_handler()
             migrate_handler_state()
 
+            from azurelinuxagent.ga.remoteaccess import get_remote_access_handler
```

Comment:
Why is this a local import?

Reply:
Just trying to match the style above with the other handlers.

---
In reply to: [194025131](https://github.com/Azure/WALinuxAgent/pull/1171#discussion_r194025131) [](ancestors = 194025131)